org.multimc.MultiMC.appdata.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Apart from upstream not being a fan of flatpak/snap/etc, please try to submit it upstream as it's also useful to regular Linux distributions.